docs/source/optim.rst

```diff
@@ -133,6 +133,24 @@ How to adjust Learning Rate
 rate based on the number of epochs. :class:`torch.optim.lr_scheduler.ReduceLROnPlateau`
 allows dynamic learning rate reducing based on some validation measurements.
 
+Learning rate scheduling should be applied after optimizer's update; e.g., you
+should write your code this way:
+
+    >>> scheduler = ...
+    >>> for epoch in range(100):
+    >>>     train(...)
+    >>>     validate(...)
+    >>>     scheduler.step()
+
+.. warning::
+  Prior to PyTorch 1.1.0, the learning rate scheduler was expected to be called before
+  the optimizer's update; 1.1.0 changed this behavior in a BC-breaking way.  If you use
+  the learning rate scheduler (calling `scheduler.step()`) before the optimizer's update
```

Comment:
I have only one minor nit. In rst, to write a code text, use double ticks. :)

Reply:
I think both work. OK I changed to double ticks in the my change and in the previous docstring.